server/src/test/java/org/elasticsearch/common/joda/JavaJodaTimeDuellingTests.java

```diff
@@ -64,11 +62,22 @@ public void testTimeZoneFormatting() {
         formatter3.parse("20181126T121212.123-0830");
     }
 
-    public void testCustomTimeFormats() {
-        assertSameDate("2010 12 06 11:05:15", "yyyy dd MM HH:mm:ss");
-        assertSameDate("12/06", "dd/MM");
-        assertSameDate("Nov 24 01:29:01 -0800", "MMM dd HH:mm:ss Z");
-    }
+    // this test requires tests to run with -Djava.locale.providers=COMPAT in order to work
```

Comment:
Should we then remove this?

Reply:
it took me a long time to find out that we run tests differently to prod regarding the `java.locale.providers` property. This was my  reminder that this is the case. Happy to remove it, was just here as a reminder for the next person :-)

Reply:
Then let's keep the reminder.